README.md

```diff
@@ -48,4 +61,18 @@ For more details, check the docstring of the `element-electrode-localization`:
 
 ### Example usage
 
-See the [workflow-array-ephys project](https://github.com/datajoint/workflow-array-ephys) for an example usage of this Element.
\ No newline at end of file
+See the [workflow-array-ephys project](https://github.com/datajoint/workflow-array-ephys) for an example usage of this Element.
```

Comment:
Future PRs should also link to the [relevant notebook](https://github.com/datajoint/workflow-array-ephys/blob/event/notebooks/08-electrode-localization.ipynb) - although this notebook is currently limited to the `event` 'dev branch'. The inclusion of localization notebook in the workflow is not transparent from the repo README.